src/plotter/presets.py

```diff
@@ -135,7 +141,20 @@ def add_and_plot(
 
         self.ratioPad.plot_histos()
 
-    def remove_empty(self):
+        self.update_ranges()
+
+    #       self.ratioPad.update_range()
+
+    def update_ranges(self):
+
+        if self.nonEmpty and not self.custom_xrange:
+            (xmin, xmax) = self._xrange_emptysupressed()
```

Comment:
Dont see how `xmin` and `xmax` is used? Shouldn't `self.mainPad.set_xrange(xmin, xmax)` be called or something?

Reply:
yes, it got dropped after refactoring